README.md

```diff
@@ -204,11 +204,22 @@ There are the following two flash modes:
 - 1: clone disk image from the disk the system is currently running to another disk of the system
 - 2: flash disk image from network to same disk the system is currently running
 
+Depending on omnect-os bootloader (uboot or grub) the flash modes are triggered differently.

```

Comment:
mix of "omnect-os bootloader" and "OS bootloader"

Reply:
changed but this was by purpose